substrate/frame/contracts/fixtures/build.rs

```diff
@@ -203,6 +203,7 @@ fn invoke_wasm_build(current_dir: &Path) -> Result<()> {
 
 	let build_res = Command::new(env::var("CARGO")?)
 		.current_dir(current_dir)
+		.env("CARGO_TARGET_DIR", current_dir.join("target").display().to_string())
```

Comment:
shouldln't we just clear it if it set, instead of setting it ? 
or the fixtures will end up here instead of OUT_DIR?

Reply:
I haven't thought of it right away, but clearing should work as well

Reply:
It has to end up wherever you expect to find them in the rest of your script. In this case `current_dir/target` is the same as the default. Otherwise the CI would fail. So deleting should have the same effect. But let's find out why clearing the environment doesn't work.

Reply:
let's stick with the current fix

if we wish to clear env, This works as well

```rust
﻿		.envs([
			("PATH", env::var("PATH").unwrap_or_default()),
			("CARGO_HOME", env::var("CARGO_HOME").unwrap_or_default()),
			("RUSTC", env::var("RUSTC").unwrap_or_default()),
		])
```